charts/astronomer/templates/astro-ui/astro-ui-ingress.yaml

```diff
@@ -86,8 +68,11 @@ metadata:
     heritage: {{ .Release.Service }}
   annotations:
     {{- if .Values.global.authSidecar.enabled  }}
-{{- toYaml .Values.global.extraAnnotations | nindent 4 }}
+      {{- toYaml .Values.global.extraAnnotations | nindent 4 }}
     {{- else }}
+    {{- if .Values.global.extraAnnotations }}
+{{- toYaml .Values.global.extraAnnotations | nindent 4 }}
+    {{- end }}
```

Comment:
@pgvishnuram was this omitted deliberately? This was the only ingress block that did not contain this clause.

Reply:
I think we forgot looking for perhost ingress logic and totally missed to include there